src/libs/OptionsListUtils.js

```diff
@@ -603,6 +603,12 @@ function getOptions(
         };
     }
 
+    // We're only picking personal details that have logins set
+    // This is a temporary fix for all the logic that's been breaking because of the new privacy changes
+    // See https://github.com/Expensify/Expensify/issues/293465 for more context
+    // eslint-disable-next-line no-param-reassign
```

Comment:
Is there any follow up issues to address the temp fix?

Reply:
Yep, we just recently merged the PR that handles when you try to chat with someone whose login you don't have in Onyx, so we should be able to undo this now. I'll create an issue for a contributor to do it.

Reply:
Or maybe actually I can just revert this.